### ISSUE TITLE
recognition done but except 1st set not correct

### ISSUE THREAD

**Issue by msnagar** [olya-d/eigenface#1]
dear olya-d,
I was trying to check output of your code but some how i didn't get the same result that you have mentioned. please see the screenshot attached with this mail.
![Screenshot from 2021-06-17 14-44-47](https://user-images.githubusercontent.com/51476142/122370260-8e95ac00-cf7c-11eb-82ae-aa9c579c5add.png)

**Comment by olya-d** [comment]
Dear @msnagar,

default values in config.cpp are different from those mentioned in the [User Guide](https://github.com/olya-d/eigenface/wiki/User-guide) (which I admit is very confusing). Once you set them exactly as in the User Guide, the output will match.

**Comment by msnagar** [comment]
Thank you,
It works perfectly as you mentioned in User Guide. Thank you again.
![Screenshot from 2021-06-25 15-47-11](https://user-images.githubusercontent.com/51476142/123410352-d9db3a80-d5cc-11eb-984c-3a6441a8a015.png)

**Comment by msnagar** [comment]
for below mentioned configuration,  I got single fault recognition in the result. I attached the screenshot of the same. 

const int Faces = 40;
const int Samples = 10;
const int Width = 92;
const int Height = 112;
const int Eigenfaces = 30;
const std::string DataPath = "faces/";
const int N = Faces; // Total number of images
const int M = Width*Height; // Resolution
const std::string SampleName = "10";
const int MaxValue = 255;

![Screenshot from 2021-06-25 16-29-43](https://user-images.githubusercontent.com/51476142/123415506-fe3a1580-d5d2-11eb-8777-2b940d27114e.png)

Again I want to say thank you for your support.
